Sources/Rendering/Core/AbstractMapper/index.js

```diff
@@ -5,15 +5,20 @@ import macro from 'vtk.js/Sources/macros';
 // ----------------------------------------------------------------------------
 
 function vtkAbstractMapper(publicAPI, model) {
+  model.classHierarchy.push('vtkAbstractMapper');
   publicAPI.update = () => {
     publicAPI.getInputData();
   };
 
   publicAPI.addClippingPlane = (plane) => {
     if (plane.getClassName() !== 'vtkPlane') {
-      return;
+      return false;
```

Comment:
Your check prevents a possible `vtkPlane` subclass. You could check `plane.isA('vtkPlane')`.
But I guess, you should let typescript enforce the input parameter type instead of checking it yourself.

Reply:
Good point but that was from before.